README.md

```diff
@@ -271,6 +271,18 @@ The last two lines are equivalent as if the variable were substituted for its va
 
 In any expression location, an object containing a single key beginning with "Fn::" calls a built-in function.
 
+##### `Fn::ToBase64`
+
+This function encodes a UTF-8 string with the base64-encoding using the [standard base-64 encoding](https://pkg.go.dev/encoding/base64#pkg-variables). **This will fail if the result is not a valid UTF-8 string**
```

Comment:
When is it possible that the result is not a valid base64 string?

Reply:
Whoops, that was my carelessness with copy-paste!